### ISSUE TITLE
Automatic approach

### ISSUE THREAD

**Issue by gingerbeardman** [kawamataryo/sky-follower-bridge#28]
I'm following almost 5000 people, since joining twitter near the beginning.

So I'd appreciate a more automated way to add people i'm following. Even if it just automates the clicking of following and "find more" and I don't mind waiting for it to complete.

**Comment by kawamataryo** [comment]
I am developing that feature now. Please wait for its release.

**Comment by gingerbeardman** [comment]
Arigatou!

**Comment by cooljeanius** [comment]
> I'm following almost 5000 people, since joining twitter near the beginning.

Coincidentally, 5000 is also the follow limit... have you hit it yet? I've been perpetually stuck at it since like 2018 (although I might finally clear my backlog soon, with so many people leaving the site recently...)

**Comment by gingerbeardman** [comment]
Yes, I am constantly up against the follow limit. But only since the last couple of months, which is pretty good for 17 years!

**Comment by gingerbeardman** [comment]
Did you make any progress with this?

**Comment by kawamataryo** [comment]
I have made some progress on this issue and it looks like we will be able to release it within this month. The video below provides a preview of the current UI design (please note this is subject to potential changes).

https://github.com/kawamataryo/sky-follower-bridge/assets/11070996/f328106d-2a5d-4a70-b094-1d8a4dbe05d9

**Comment by kawamataryo** [comment]
This is a feature that, once you press the search button, will scrutinize all followers to detect similar users of Bluesky without the need to press the Find more button.

The Follow buttons must be pressed manually one by one. This is because there may be false positives in the detection results.

**Comment by gingerbeardman** [comment]
Oh, this is not what I was looking for. I would need to do 5000 clicks!

**Comment by cooljeanius** [comment]
I think the issue of false positives is an important one to consider, so the "half solution" of automating the "find more" part still ought to be worthwhile for now, until the problem of how to handle the false positives can be resolved.

**Comment by kawamataryo** [comment]
I see. If there is a request to ignore false positives and follow in bulk, I will try to accommodate this after releasing this feature. Once this functionality is in place, it should not be too difficult.

**Comment by gingerbeardman** [comment]
If I could do one click to add all the exact matches.

Afterwards, perhaps then it could show me a list of only the false matches - a smaller list - which I would then be in a better position to go through manually. But, I might leave it for another day so it would be good if it could get back to that list of all potential false positives in some quick way.

**Comment by MagicalDrizzle** [comment]
Hi! it seems like this feature is not implemented on Firefox? Actually, Firefox version has been lagging behind for quite a while now...(v0.7.5 vs v0.9.1)

**Comment by gingerbeardman** [comment]
How's it going with this one?

**Comment by samolukadjo** [comment]
Why's the Firefox version of the extension lagging behind?

**Comment by cooljeanius** [comment]
> Why's the Firefox version of the extension lagging behind?

I can't claim to speak for this specific extension, but I can say that in general, the Firefox review process for add-ons is often slower; we've been having some trouble with it for BlueBlocker, too: https://github.com/kheina-com/Blue-Blocker/discussions/327

**Comment by samolukadjo** [comment]
> > Why's the Firefox version of the extension lagging behind?
> 
> I can't claim to speak for this specific extension, but I can say that in general, the Firefox review process for add-ons is often slower; we've been having some trouble with it for BlueBlocker, too: [kheina-com/Blue-Blocker#327](https://github.com/kheina-com/Blue-Blocker/discussions/327)

I noticed this on blue blocker as well, actually. I wonder what it could be? Whenever I upload an update to my super simple calculator extension, it gets pushed in seconds. Maybe the human review staff are understaffed so when an extension is flagged it gets put on hold for very long?

**Comment by cooljeanius** [comment]
> > > Why's the Firefox version of the extension lagging behind?
> > 
> > 
> > I can't claim to speak for this specific extension, but I can say that in general, the Firefox review process for add-ons is often slower; we've been having some trouble with it for BlueBlocker, too: [kheina-com/Blue-Blocker#327](https://github.com/kheina-com/Blue-Blocker/discussions/327)
> 
> I noticed this on blue blocker as well, actually. I wonder what it could be? Whenever I upload an update to my super simple calculator extension, it gets pushed in seconds. Maybe the human review staff are understaffed so when an extension is flagged it gets put on hold for very long?

Seems to be a common issue with the add-on review process; it also affected uBlock Origin Lite recently: https://www.pcworld.com/article/2474353/popular-ad-blocker-removed-from-firefox-extension-store.html

**Comment by kawamataryo** [comment]
We've added an "Follow All" button, but it's only available on the Chrome version starting from v1.0.0. Please give it a try!  
(Firefox is still pending review.)


![CleanShot 2024-11-07 at 20 10 13@2x](https://github.com/user-attachments/assets/95f98c12-d5ea-47dc-8339-973247657cb8)

**Comment by cooljeanius** [comment]
Cool, hopefully the Firefox review completes soon!

**Comment by gingerbeardman** [comment]
Thanks! I use Chrome just for this extension. 

Safari is my main browser.
